datasets.toml

```
@@ -1,6 +1,18 @@
 version=1.0
 
 
+[[datasets]]
+  # The doi is invalid, but the paper is indexed in OpenAlex
+
+  key="Giesen_2021"
+
+  [datasets.publication]
+  url="https://www.gavinpublishers.com/article/view/overview-of-pain-interventions-for-hospital-and-community-care-nurses-a-systematic-scoping-review"
```

```suggestion
doi="10.29011/26889501.101265"
  url="https://www.gavinpublishers.com/article/view/overview-of-pain-interventions-for-hospital-and-community-care-nurses-a-systematic-scoping-review"
```

Let's add the DOI anyway. They will most likely update the metadata behind the doi. As long as the OpenAlex pipeline works, I'm happy.